packages/manager/modules/pci/src/projects/project/storages/databases/database/logs/logs.scss

```diff
@@ -0,0 +1,46 @@
+ovh-manager-pci-storages-database-logs-component {
+  $logger-background-color: white;
+  $logger-text-color: black;
+
+  .logger {
+    background: $logger-background-color;
+    padding: 0rem 0rem;
+    margin: 0rem 0;
+    color: $logger-text-color;
+    height: 65vh;
+    box-sizing: border-box;
+  }
+
+  .logger_placeholder {
+    text-align: center;
+    margin-top: 170px;
+    color: #999;
+  }
+
+  .logger__entry-message {
+    display: inline-block;
+    line-height: 20px;
+    overflow-wrap: break-word;
+    white-space: pre-wrap;
+    letter-spacing: 0.14px;
+    font-family: 'Ubuntu Mono', monospace;
+    font-size: 0.85rem;
+  }
+
+  tail-logs p {
+    margin: 0;
+    font-size: medium;
+  }
+  tail-logs p.debug {
+    color: green;
+  }
+  tail-logs p.warning {
+    color: wheat;
+  }
+  tail-logs p.info {
+    color: white;
+  }
+  tail-logs p.error {
+    color: red;
```

Comment:
Not possible to use https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-colors--primary

Reply:
Done here: https://github.com/ovh/manager/pull/5431/commits/3b6e0c14c882cf92c9e3645376cd5c85da158551